tests/integration-tests/tests/create/test_create/test_create_disable_sudo_access_for_default_user/pcluster_update_login_nodes_count_to_0.config.yaml

```diff
@@ -0,0 +1,31 @@
+Image:
+  Os: {{ os }}
+LoginNodes:
+  Pools:
+    - Name: login
+      InstanceType: t2.micro
```

Comment:
Can we use `{{ instance }}` instead of hard-coding t2.micro?
RHEL 9 seems not to be running well on `micro` instances.
Same comment for other configs